frontend/src/i18n/messages/en/index.ts

```diff
@@ -47,6 +47,7 @@ export const messages = {
       hide_summary_description: "Hide full description",
       nofo_download: "Download Full Announcement (NOFO)",
       jump_to_documents: "Jump to all documents",
+      zip_download: "Download All Documents as ZIP file",
```

Comment:
is this the real copy? Are there designs for this? @michellemin-nava @crystabelrangel